src/main/java/ml/codeboy/bukkitbootstrap/gui/Gui.java

```diff
@@ -71,6 +71,10 @@ public void onDrag(InventoryDragEvent event) {
             event.setCancelled(true);
     }
 
+    public void addItem(ItemStack item, AdvancedAction action) {
+        addItem(item, (Action) action);
+    }
+
```

Comment:
I don't think that's necessary thanks to Action being super class of AdvancedAction